src/coreclr/System.Private.CoreLib/CreateRuntimeRootILLinkDescriptorFile.targets

```diff
@@ -14,7 +14,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <_ILLinkDescriptorsFilePaths Include="$(ILLinkDirectory)ILLink.Descriptors.xml" />
+    <_ILLinkDescriptorsFilePaths Include="$(ILLinkDirectory)ILLink.Descriptors.Common.xml" />
```

Comment:
I'd call this `Shared.xml`, we use that term elsewhere. e.g.

https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.Shared.xml

Reply:
Or... maybe just leave it `ILLink.Descriptors.xml`, and also have `ILLink.Descriptors.Windows.xml`.